Makefile

```diff
@@ -0,0 +1,55 @@
+SHELL = /bin/bash
```

Comment:
In addition to automation, this make file also serves as directives for building the code.